README.md

```diff
@@ -282,11 +282,22 @@ encoding of the request. The parsing can be aborted by throwing an error.
 ## Errors
 
 The middlewares provided by this module create errors depending on the error
-condition during parsing. The errors will typically have a `status`/`statusCode`
-property that contains the suggested HTTP response code, an `expose` property
-to determine if the `message` property should be displayed to the client, a
-`type` property to determine the type of error without matching against the
-`message`, and a `body` property containing the read body, if available.
+condition during parsing. 
+
+This is passed to an Express error handling middleware, if one is defined, 
```

Comment:
Is this necessary to say? This is how all express middleware work and it would be nice if I didn't need to add this paragraph to every middleware module's readme. Is there is reason that would lead one to believe that errors from middleware are not typically passed to express error handling middleware? This module is usable outside express as well, so if this is to be kept, it likely needs to better explain how the errors are passed for the non express users (right now it is omitted as one would refer to their frameworks docs on handling middleware errors).

Reply:
My reasoning behind including this, is that body-parser is one of the more popular packages related to Express, and it's therefore possible that body-parser users aren't Express experts who are looking to configure error handling, and mistakenly think that body-parser is the place to configure it. 

Perhaps it'd be more useful to add a link to the original Express documentation regarding error handling middleware?

This way it'd retain its purpose for non-expert users who are looking here for information, while keeping the onus of documenting error handling on Express.

Reply:
I don't think Express users will end up here as the primary method to parse bodies, as express has this built into itself, as express.json, express.urlencoded, etc.

Reply:
I take your point. Happy to remove this section in thiscase.